Strategy11/formidable-forms
# Add subscription limit setting and a feature to auto-cancel a subscription after it reaches the payment limit

Syncs changes in https://github.com/Strategy11/formidable-stripe/pull/330

<img width="1284" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/41271840/0c4fb937-f534-4ab9-8e70-a07a10c698f1">

### Test steps
1. Go to form actions and add a Payment action
2. Set the Payment type to 'recurring'
3. Confirm that a new setting appears that allows configuring the payment limit
4. Configure all the required settings and save.
5. Confirm that the subscription is cancelled after the limit is hit
